src/R/Editor/Impl/Data/RSessionDataObject.cs

```diff
@@ -63,12 +64,33 @@ public class RSessionDataObject : IRSessionDataObject {
                 Value = promiseInfo.Code;
                 Class = TypeName = "<promise>";
             } else if (DebugEvaluation is IRActiveBindingInfo) {
-                const string ActiveBindingValue = "<active binding>";
+                const string ActiveBindingText = "<active binding>";
                 var activeBinding = (IRActiveBindingInfo)DebugEvaluation;
+                var activeBindingValue = activeBinding.Value;
 
-                Value = ActiveBindingValue;
-                TypeName = ActiveBindingValue;
-                Class = ActiveBindingValue;
+                if(activeBindingValue != null) {
```

Comment:
The code inside mostly duplicates `IRValueInfo` branch above - this should probably be refactored into a reusable method. Or maybe reorder the branches so that we do checks for promises and active bindings first, and then check the value, so that the first two branches can tweak the input for the third. E.g.:
```cs
IRValueInfo valueInfo = DebugEvaluation as IRValueInfo;

if (DebugEvaluation is IRPromiseInfo) {
  ...
} else if (DebugEvaluation is IRActiveBindingInfo) {
  ...
  valueInfo = activeBinding.Value;
  ...
}

if (valueInfo != null) { ... }
```